test/ctr.bats

```diff
@@ -871,6 +871,9 @@ function teardown() {
 }
 
 @test "run ctr with image with Config.Volumes" {
+	if test -n "$UID_MAPPINGS"; then
+		skip "userNS enabled"
+	fi
```

Comment:
Once cri supports getting the UID_MAPPINGS from CRI-O this test should be turned back on, correct?

Could we chown the volume from the host to work with the UID_MAPPINGS?